ui/lib/kv/addon/routes/create.js

```diff
@@ -22,6 +22,7 @@ export default class KvSecretsCreateRoute extends Route {
       backend,
       path,
       secret: this.store.createRecord('kv/data', { backend, path }),
+      metadata: this.store.createRecord('kv/metadata', { backend, path }),
```

Comment:
We should also rollback `model.metadata` if dirty on leave right? (withConfirmLeave)